canonicalwebteam/blog_extension/api.py

```diff
@@ -0,0 +1,123 @@
+import os
+
```

Comment:
As discussed, could we rename `blog_extension` to `blog`?

Reply:
done